src/OpenTelemetry.ClrProfiler.Managed/Configuration/IntegrationSettings.cs

```diff
@@ -21,8 +21,7 @@ public IntegrationSettings(string integrationName, IConfigurationSource source)
                 return;
             }
 
-            Enabled = source.GetBool(string.Format(ConfigurationKeys.Integrations.Enabled, integrationName)) ??
-                      source.GetBool(string.Format("OTEL_{0}_ENABLED", integrationName));
+            Enabled = source.GetBool(string.Format(ConfigurationKeys.Integrations.Enabled, integrationName));
```

Comment:
Did we intentionally remove the fallback setting check? If so it's not clear why we're removing it as part of this PR.

Reply:
When reading Chris's comment I was mixing this up with `OTEL_DOTNET_AUTO_{0}_ENABLED` (previously `OTEL_TRACE_{0}_ENABLED`). While a pure refactoring PR would be nice, it does make sense to remove the second undocumented key so I'm OK with including this in the PR